app/kubernetes-ingress-controller/preserve-client-ip.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Preserve Client IP"
+
+description: |
+  How to pass the client IP through Gateway to the upstream service
+
+content_type: reference
+layout: reference
+
+products:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+---
+
+
+Kong is deployed using a Kubernetes Service of type `LoadBalancer`. This can result in the loss of the actual Client IP address, and lead to Kong observing the IP address of the Load Balancer as the client IP address. This guide lays out different methods of solving this problem.
```

Comment:
```suggestion
{{site.base_gateway}} is deployed using a Kubernetes Service of type `LoadBalancer`. This can result in the loss of the actual client IP address, and can lead to {{site.base_gateway}} observing the IP address of the Load Balancer as the client IP address. This guide lays out different methods of solving this problem.
```